src/app/modals/update-name-modal/update-name-modal.component.html

```diff
@@ -0,0 +1,11 @@
+<h1 mat-dialog-title>Hello {{data.displayName}}</h1>
```

Comment:
Since we show the `displayName` in the title, let's copy it and use it here so that when the user modify their name it doesn't change in the title as well.

Reply:
Come on... that's a cool feature! 😁